javascript/organizing_your_javascript_code/factory_functions_and_module_pattern.md

```diff
@@ -75,14 +75,14 @@ console.log(add5(2)) // logs 7
 
 A lot going on, so let's break it down:
 
-1. The `makeAdding` function takes an argument, `first`, and returns another **function** (which we have named as `add5` while using).
-1. When an argument is passed to the returned function (in our case, `add5`), it returns the result of adding up the number passed earlier to the number passed now (`first` to `second`).
+1. The `makeAdding` function takes an argument, `firstNumber`, and returns another **function**.
```

Comment:
If you're going to rename this variable, you must also rename all its references in subsequent paragraphs. I'd suggest just leaving it `first`.

Reply:
Looks like this was dealt with in PR #26661 and was merged; I missed it when I created this PR.